.github/workflows/platform-api-cloud-release.yml

```
@@ -0,0 +1,43 @@
+          echo "IMAGE_VERSION=${BRANCH}-${COMMIT}" >> "$GITHUB_OUTPUT"
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.26.5'
+          cache: false
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          install: true
```

actionlint (1.7.12)

[error] avoid using deprecated input "install" in action "docker/setup-buildx-action@v3": "docker buildx install" command is deprecated and will be removed in a future release, use BUILDX_BUILDER environment variable instead

(action)